app/src/main/java/ng/com/thewhitecellfoundation/haemcam/ui/home/FeedbackFragment.kt

```diff
@@ -15,7 +19,38 @@ import ng.com.thewhitecellfoundation.haemcam.databinding.FragmentFeedbackBinding
 class FeedbackFragment : Fragment(R.layout.fragment_feedback) {
 
     private val binding by viewBinding(FragmentFeedbackBinding::bind)
+
+    @SuppressLint("ResourceType")
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
+        val categories = resources.getStringArray(R.array.categories)
+        val arrayAdapter = ArrayAdapter(requireContext(), R.layout.fragment_feedback_dropdown_item, categories)
+        binding.fragmentFeedbackAutocompleteTextview.setAdapter(arrayAdapter)
+
+        val name = "Dr Anthonia Peterson"
+        val message = "Here are some medical feedback.... " + "Here are some medical feedback....Here are some medical feedback....  " + " Here are some medical feedback...."
```

Comment:
Do not hard code the ellipsis(...) at the end of text.

Reply:
Okay sir. Making corrections now